src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -186,6 +186,7 @@ const DOCS: {[key: string]: DocCategory[]} = {
         {id: 'progress-spinner', name: 'Progress spinner',
             examples: ['progress-spinner-configurable']},
         {id: 'progress-bar', name: 'Progress bar', examples: ['progress-bar-configurable']},
+        {id: 'ripple', name: 'Ripples', examples: ['ripple-overview']}
```

Comment:
@jelbourn This PR has now merge conflicts because it needs a `summary`. Do you have any summary in mind?